x-pack/plugins/licensing/server/license_feature.test.ts

```diff
@@ -4,7 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { ILicense } from './types';
+import { ILicense } from '../common/types';
```

Comment:
Move file to `../common`?

Reply:
We *could*, but it would complicate the test since the setup is different for the server vs. the client.